Makefile.toml

```diff
@@ -15,7 +15,7 @@ workspace = false
 [tasks.polkadot]
 script = '''
 if [ ! -d "../polkadot" ] ; then
-    git clone -b v0.9.18 https://github.com/paritytech/polkadot ../polkadot
+    git clone -b v0.9.22 https://github.com/paritytech/polkadot ../polkadot
```

Comment:
we use `https://github.com/composableFi/polkadot` in composable-sandbox
```
git clone -b mmr-polkadot-v0.9.22 https://github.com/composableFi/polkadot ../polkadot
```

Reply:
hm, not sure. are not our tests to run default polkadot integration? as polkadot on rococo and kusama is not our?

Reply:
will close for now as I improve simple version, not whole thing.